.github/workflows/go-e2e.yml

```diff
@@ -16,7 +16,7 @@ on:
       docker_image_prefix:
         description: 'Docker image prefix'
         required: true
-        default: 'ghcr.io/timescale/dev_promscale_extension:master-ts2'
+        default: 'ghcr.io/timescale/dev_promscale_extension:<branch>-ts2'
```

Comment:
@JamesGuthrie how does this work?

Reply:
When you want to activate a manual workflow run, then it's always against a specific branch name. You need to put the branch that you want to run against in place of `<branch>`.

Reply:
and it will be removed anyway when #1402 lands 😅